packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.routes.js

```diff
@@ -10,11 +7,9 @@ angular.module('Billing').config(($stateProvider) => {
         squash: true,
       },
     },
-    template,
-    controller,
-    controllerAs: '$ctrl',
+    component: 'billingHistory',
     translations: {
-      value: ['../history', './postalMailOptions'],
+      value: ['./postalMailOptions'],
```

Comment:
Sure this was not used ?

Reply:
`../history` in that context is equivalent to '.'. It has been injected differently https://github.com/ovh/manager/pull/2894/files/658c3261fbad3c91d2f3cb480ca0d37868330f49#diff-ca5078e6049a66495c3581e05d212608R14